src/server.js

```diff
@@ -90,18 +90,25 @@ const marketplaceRankPayloadBlock = (marketplaceData) => {
  * @return  {object}  Slack payload block object.
  */
 const issueCreditPayloadBlock = (orgDrupalIssueCreditCount) => {
-  const creditCountMax = datastore.variableGet(config.keyValueDefaults.issueCreditCountMaxVarKey);
+  const orgDrupalIssueCreditCountInt = parseInt(orgDrupalIssueCreditCount, 10);
```

Comment:
TIL `parseInt` does not presume the radix to be 10. 🤯

Reply:
Thanks to eslint for flagging that one!